app/src/main/java/org/schabi/newpipe/util/ShareUtils.java

```diff
@@ -64,6 +65,11 @@ public static void openUrlInBrowser(final Context context, final String url,
             // No browser set as default (doesn't work on some devices)
             openInDefaultApp(context, intent);
         } else {
+            if (defaultPackageName.isEmpty()) {
+                // No app installed to open a web url
+                Toast.makeText(context, R.string.no_app_to_open_intent, Toast.LENGTH_LONG).show();
+            }
+
             try {
                 intent.setPackage(defaultPackageName);
                 context.startActivity(intent);
```

Comment:
Toast is good!
But here we tell NewPipe to open the stream we just tried to open in a browser:

https://user-images.githubusercontent.com/17365767/104925439-20ce5f80-599f-11eb-96a1-8a925acc9307.mp4

The black flickering is caused by Android Studio. Seems like recording is somewhat broken